src/server-dev.ts

```diff
@@ -10,15 +11,23 @@ process.nextTick(async () => {
                 db: process.env.DATABASE_URL
                     ? undefined
                     : {
-                          user: 'unleash_user',
-                          password: 'password',
-                          host: 'localhost',
-                          port: 5432,
-                          database:
-                              process.env.UNLEASH_DATABASE_NAME || 'unleash',
-                          schema: process.env.UNLEASH_DATABASE_SCHEMA,
+                          user: process.env.DATABASE_USERNAME || 'unleash_user',
+                          password: process.env.DATABASE_PASSWORD || 'password',
+                          host: process.env.DATABASE_HOST || 'localhost',
+                          port: parseEnvVarNumber(
+                              process.env.DATABASE_PORT,
+                              5432,
+                          ),
+                          database: process.env.DATBASE_NAME || 'unleash',
+                          schema: process.env.DATABASE_SCHEMA,
                           ssl: false,
-                          applicationName: 'unleash',
+                          applicationName:
+                              process.env.DATABASE_APPLICATION_NAME ||
+                              'unleash',
+                          disableMigration: parseEnvVarBoolean(
+                              process.env.DATABASE_DISABLE_MIGRATION,
```

Comment:
It would be better to handle migration flag through environment variable in dev mode because sometimes users don't want to run migration and they have to turn it off by fixing code if they can't handle it via environment variable.